### ISSUE TITLE
[Bug]: Arial32 font dependency broken ?

### ISSUE THREAD

**Issue by ghost** [bottlesdevs/Bottles#3144]
### Describe the bug

I installed the Arial32 Dependency inside Bottles, it should put a arial.ttf file inside C:\Windows\Fonts, but it did not put anything there.

### To Reproduce

1. Install Arial32 from Dependencies tab
2. Navigate to C:\Windows\Fonts inside the Bottles
3. No arial.ttf file

### Package

Flatpak from Flathub

### Distribution

Fedora Silverblue 39

### Debugging Information

```shell
Official Package: true
Version: '51.9'
DE/WM: gnome
Display:
    X.org: true
    X.org (port): :0
    Wayland: true
Graphics:
    vendors:
        intel:
            vendor: intel
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.5.6-300.fc39.x86_64
Disk:
    Total: 3039555584
    Free: 3039399936
RAM:
    MemTotal: 5.7GiB
    MemAvailable: 3.4GiB
Bottles_envs: null
```

### Additional context

Looking at the manifest for the font, it is supposed to download https://mirrors.kernel.org/gentoo/distfiles/5e/arial32.exe and cab_extract it.

I went to the link and it did not have anything, i then went to 1 directory above it and Ctrl+F'd "Arial" and still found nothing.

I think the URL the manifest was supposed to get the .exe from simply does not have the .exe anymore.

**Comment by xioren** [comment]
Yes, there is a [fix](https://github.com/bottlesdevs/dependencies/pull/204) waiting to get mearged.

**Comment by rodhfr** [comment]
18:43:20 (ERROR) Failed to download [https://cfhcable.dl.sourceforge.net/project/corefonts/the%20fonts/final/arial32.exe] 
18:43:20 (ERROR) Failed to install dependency: Microsoft Arial Font

**Comment by macccorehenni** [comment]
I have the same issue as @rodhfr on Debian 12 as of today. Is this fixed?

**Comment by Lastdough** [comment]
![Screenshot from 2024-08-07 22-16-05](https://github.com/user-attachments/assets/78dab2b5-290c-4877-bc44-823a7b3a535b)
just want to ask is this happening relate to font not detected?

I'm also on fedora 39

**Comment by polytect** [comment]
Creating a bottle:
Installing dependency: Microsoft Arial Font …

Half hour later:
Installing dependency: Microsoft Arial Font …

**Comment by runekaagaard** [comment]
> Creating a bottle: Installing dependency: Microsoft Arial Font …
> 
> Half hour later: Installing dependency: Microsoft Arial Font …

Yeah, same

**Comment by murderuo** [comment]
i got same error, any1 fix this issue ?

**Comment by murderuo** [comment]
i found something ;
https://github.com/bottlesdevs/Bottles/issues/3331#issuecomment-2480522669
